README.md

```diff
@@ -97,26 +97,33 @@ Documentation
 The documentation framework uses the popular Sphinx documentation generating engine and autodoc plugins to give
 developers the flexibility of writing Extra documentation interleaved with the autogenerated documentation created by
 autodoc.
-Documentation can be generated based on the sources in the docs/source directory. After having installed the packages in 
-requirements.txt (which is done automatically by the make command) documentation can be generated by either of the 
-following ways:
 
-# Generating documentation
+## Prerequisite: Graphviz
+
+To display the images in the documentation, you need to have [Graphviz](http://www.graphviz.org/) installed and make sure the *dot* executable is on the PATH. For example, for users of [homebrew](http://brew.sh/):
+
+    brew install graphviz
+
+## Generating documentation
+
+Documentation can be generated based on the sources in the docs/source directory. After having installed the packages in 
+requirements.txt (which is done automatically by the make command) documentation can be generated by one of the 
+following three ways:
 
-## Calling setuptools
+1. Calling setuptools
 
     python setup.py build_sphinx
     
-## Running make in the docs directory where separate makefiles and a make.bat file is giving a variety of options.
+2. Running make in the docs directory where separate makefiles and a make.bat file is giving a variety of options.
 
     cd docs
     make html
```

Comment:
Not sure why but when I preview the file this comes out looking like one line `cd docs make html` which is misleading.